content/docs/user-guide/merge-conflicts.md

```diff
@@ -139,4 +139,4 @@ outs:
   - path: data.xml
 ```
 
-And then `dvc update` the `.dvc` file.
+And then `dvc pull` the import (`dvc.xml` in the example above).
```

Comment:
Hi @efiop, can you confirm this is actually more correct?
Rationale: update may bring an even newer version and overwrite the hash you selected during merge, while pull will just make sure you have the correct version in cache and workspace.
Thanks

Reply:
@jorgeorpinel The example shows deleting the hash, so `dvc update` is more correct, as it will record the new hash.

Reply:
Hmmm true. But still then you may get a newer version which may not be the goal of this merge. I'll add a pull option and keep update as well.

Reply:
Wait no, that's probably too difficult. OK I'll just add a note the implications of updating.